client/app/styles/_base.scss

```diff
@@ -102,7 +102,7 @@ a {
 
 .overlay-wrapper {
   align-items: center;
-  background-color: fade-out($background-average-color, 0.1);
+  background-color: transparent;
```

Comment:
> @fbarl was there another reason why the overlay-wrapper would need a background colour? I remember talking about it...

Yeah, I made it non-transparent about a year ago to make overlays more readable when there is content in the background. Compare the footer text:

**now**
![image](https://user-images.githubusercontent.com/1216874/31464001-527b8ee0-aed0-11e7-9567-5d75c76f3820.png)

**before**
![image](https://user-images.githubusercontent.com/1216874/31464053-7f084566-aed0-11e7-813e-8a1dd4bb6c8a.png)

So yeah, I'd rather keep it mostly opaque, unless we figure a better solution (which we could also do in a separate PR).